website/docs/d/datasource_mdb_postgresql_cluster.html.markdown

```diff
@@ -89,6 +89,7 @@ The `user` block supports:
 * `permission` - Set of permissions granted to the user. The structure is documented below.
 * `login` - User's ability to login.
 * `grants` - List of the user's grants.
+* `conn_limit` - The maximum number of connections per user.
```

Comment:
It's better to use "limit" instead of "maximum" because we also have a max_connections setting.

Reply:
Name is from official docs https://cloud.yandex.com/docs/managed-postgresql/operations/cluster-users